internal/fs/sd_windows.go

```diff
@@ -42,7 +42,7 @@ var lowRestoreSecurityFlags windows.SECURITY_INFORMATION = windows.DACL_SECURITY
 // This needs admin permissions or SeBackupPrivilege for getting the full SD.
 // If there are no admin permissions, only the current user's owner, group and DACL will be got.
 func getSecurityDescriptor(filePath string) (securityDescriptor *[]byte, err error) {
-	onceBackup.Do(enableBackupPrivilege)
+	onceBackup.Do(EnableBackupPrivilege)
```

Comment:
If `EnableBackupPrivilege` is already called by the backup command, then this call here is redundant.